crates/bevy_ecs/src/schedule/schedule.rs

```diff
@@ -394,10 +394,23 @@ impl Schedule {
         self.graph.passes.remove(&TypeId::of::<T>());
     }
 
+    /// Returns `true` if the schedule contains this build pass. Returns `false` otherwise.
+    pub fn contains_build_pass<T: ScheduleBuildPass>(&self) -> bool {
+        self.graph.passes.contains_key(&TypeId::of::<T>())
+    }
+
     /// Changes miscellaneous build settings.
+    ///
+    /// If `settings.auto_insert_apply_deferred` is `false`, this clears `*_ignore_deferred`
```

Comment:
Can you use doc links when possible ? this ensures docs stay up to date when/if api changes

```suggestion
    /// If [`settings.auto_insert_apply_deferred`][ScheduleBuildSettings::auto_insert_apply_deferred] is `false`, this clears `*_ignore_deferred`
```